### ISSUE TITLE
Consider adding full suite of accessibility checks [feature request]

### ISSUE THREAD

**Issue by jasonday** [htmlhint/HTMLHint#221]
Consider adding full WCAG (plus best practices) accessibility checks. See [axe](https://github.com/dequelabs/axe-core) for potential rule set.

**Comment by jasonday** [comment]
bump.

**Comment by dwaxweiler** [comment]
Bump

**Comment by coliff** [comment]
I don't think we will add the full suite of accessibility checks. AXE is great, free to use, well maintained and available as VS Code extension, CLI tool and browser extension.

There are lots of useful rules we can add to HTMLHint and many requested features, I don't think it makes sense to duplicate the work of existing accessibility tools.

Having said all that, we are still adding many accessibility rules (https://htmlhint.com/changelog/), it's just that we don't intend to have full WCAG coverage.
